### PR TITLE
Adding a section about calibration check

### PR DESCRIPTION
**Pull request by susilm**

### CODE REVIEW

File: tech/career-framework/performance-reviews.md [hunk 1]
```diff
@@ -24,3 +25,14 @@ Before the performance review, **allocate 2 hours** to prepare for it, to perfor
...
+
+### Consistency Checks (Calibration)
+
+With a large organization, it is important that we are consistent in the evaluation across various families, teams and managers. At the same time, engineering leadership is performing sanity checsk at the end of each performance review cycle to identify possible anomalies that would be worth looking into. 
+
+It is responsibility of family leads to go through the following steps:
+- Add latest performance review results to `Reviews` sheet in the tech-wide payroll file. This sheet contains historical ratings so it's a data source that can be used to understand the 'delta' (change of employees' progress since previous performance review)
```
Comment (siroky) [inline tech/career-framework/performance-reviews.md:34]: I think this shouldn't be responsibility of family leads but rather of the person who prepares the calibration report - to make the snapshot. But ideally, there shouldn't be such need and the sheet should calculate it automatically, I just didn't get to it 😅

File: tech/career-framework/performance-reviews.md [hunk 2]
```diff
@@ -24,3 +25,14 @@ Before the performance review, **allocate 2 hours** to prepare for it, to perfor
...
+- Identify `calibration groups` that you want to look into - calibration group is a combination of a grade (e.g. `IC3` or `IC4`) and a track (e.g. `Engineer`). It doesn't make sense to compare career growth of employees on different tracks and also people of different seniority (e.g. `IC2` against `IC4`)
```
Comment (siroky) [inline tech/career-framework/performance-reviews.md:35]: Isn't calibration group of each person clear? That's given by their track and grade so I guess there's no work there.

File: tech/career-framework/performance-reviews.md [hunk 3]
```diff
@@ -24,3 +25,14 @@ Before the performance review, **allocate 2 hours** to prepare for it, to perfor
 ## Second Level Managers
 
 When you receive the competency scores of a given employee, update the tech-wide payroll sheet with the scores and bump the reviewed column to performance review date. In addition, **review the payroll sheet**, identify whether the employees deserve a salary increase (review the last raise date, their minimum and index) and validate the increase with the 1st level manager. Make sure the salaries are fair, and nobody is under the minimum. Good practice is to also review the personal goals of independent contributors and provide adjustment feedback to the team leads, if necessary.
+
+### Consistency Checks (Calibration)
```
Comment (siroky) [inline tech/career-framework/performance-reviews.md:29]: Maybe it would be best to split this to two parts:

- One should be describing how do we create the calibration report and what it consists of. That can be used as a documentation for the person who will be facilitating that. That's not too relevant for anyone else so I'd put it on the bottom.
- More important section should be here, under Second Level Managers, describing what to look for in the calibration report, how to interpret it, what should be the questions they should seek answers for. Kindof the last 3 points in this section, but I'd maybe describe the types of anomalies in more detail. Ideally, structure of the calibration report should be driven by these "use cases" and it should highlight the anomalies automatically.

**Review by siroky — CHANGES_REQUESTED** [review]
Actually, I think the calibration is becoming a bigger topic that would deserve its own section. Because it's another process and cadence we're doing and we should even embed it into the calendar:
- month before performance reviews, there should be pre-calibration session
- month before performance reviews, there should be the training/workshop for managers after pre-calibration session
- month after performance reviews, the should be the post-calibration session.
- optionally, month after performance reviews, there should be retrospective with managers

So my proposal is to create a new chapter in the career framework `10. Calibration`. Describe the initiatives there (pre calibration, training, post-calibration, retrospective). Embed it into the calendar as well. And refer to it from this guide as a reminder - e.g. that manager should ensure before doing perf. review that they participate/watch the training. 2nd level managers should perform pre-calibration session and inform their teams.

image

File: tech/career-framework/performance-reviews.md (hunk 2 above)
Comment (susilm) [reply tech/career-framework/performance-reviews.md]: I rephrased this a little in context of other changes. But the point was that I want to explain what `calibration group` term means as we are using this term in the subsequent sentences.

File: tech/career-framework/performance-reviews.md (hunk 1 above)
Comment (susilm) [reply tech/career-framework/performance-reviews.md]: This was in the end rephrased in context of larger changes. But also, details about Mews-specific implementation of spreadsheet probably shouldn't be included in this framework.

**Comment by susilm** [comment]
Uploading new version based on discussions during performance process workshops with other managers and directors. Some highlights:
* introducing calibration event during the performance review cycle - managers propose ratings and there is a calibration meeting to bless the proposed score before it gets communicated to employees. It prolongs the process a bit but should be worth it.  see `calibration.md` for details
* I didn't want to overcomplicate calendar so I rather added initial draft of timeline to describe "what happens within each performance review cycle". Eventually we can extend it and also share more visual version with specific dates for each ritual before reach review cycle.

File: tech/career-framework/calibration.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Calibration (Consistency Checks)
+
+With a large organization, it is important that we are consistent in the evaluation across various families, teams and managers. To achieve consistency across the board, there are several routines that take place during the performance review cycle:
+- Calibration sessions trainings for Managers
```
Comment (siroky) [inline tech/career-framework/calibration.md:4]: The wording seems confusing - I'd use either "calibration trainings" or "calibration sessions" but not "calibration sessions trainings".

File: tech/career-framework/calibration.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+- Pre-review Calibration
+- Data-based Sanity Check
+- Lessons Learned
+	
+  
+## Training - Calibration sessions trainings for Managers
+
+Having people aligned on expectations from each role up front is the most efficient way. We are going to have a sessions with managers before each performance review cycle to make sure everyone has the same understanding of:
```
Comment (siroky) [inline tech/career-framework/calibration.md:12]: ... have a session or have sessions

File: tech/career-framework/calibration.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+- Impact expectations from each Level (Beginner, Intermediate, Advanced, Expert, Leading Expert) - especially for `Expert` and `Leading Expert` level there is sometimes confusion what is the expected impact of relevant competencies 
+- Are we looking for these expectations as grow aspect or performance (role responsibility) aspect? Competencies once achieved are not guaranteed to be retained for ever
```
Comment (siroky) [inline tech/career-framework/calibration.md:14]: - Missing dot at the end of sentence.
- Not sure I understand "Are we looking for these expectations as grow aspect or performance (role responsibility) aspect?" maybe rewording would help or clarifying what "these expectations" refer to.

File: tech/career-framework/calibration.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+- What should be the manager's action if people are under-performing or over-performing.
+- Make sure everyone is aware of additional resources - Examples of Competency Interpretations. Opportunity to discuss questions within larger group to get more opinions.
+		
+    
+## Pre-review Calibration
+
+This step helps us to catch possible inconsistencies early in the cycle, before the final performance ratings would be communicated to employees. We want to encourage in-person discussion about proposed ratings and keep these calibration discussions within groups of managers managing similar groups of people.
+
+The process is following:
+- Employees fill in their self-rating in the competency spreadsheet and submit it to their manager AHEAD of the performance review meeting. Employees are encouraged to ask their colleagues for feedback before submitting the self-review.
```
Comment (siroky) [inline tech/career-framework/calibration.md:24]: This should be reflected in the performance review process somewhere around here: https://github.com/MewsSystems/org/pull/31/files#diff-4ed7da08b1010b2a67cf6b15a8300df8bfd2ae07d7d7f00b625672f656b15df8R5

File: tech/career-framework/calibration.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+- Managers prepare draft of the performance reviews for the employees based on their self-evaluation, feedback collected and performance observed over the last 6 months.
+- Managers submit the proposed score to their superiors (eg. Engineering Managers submit proposed score of their ICs to Engineering Directors). The draft score is added to the global spreadsheet.
+- Pre-review calibration sessions are organized within various groups:
+  - For each family, its Engineering Director and all Engineering Managers meet and go through:
+    - For all promotions to senior roles and higher (eg. to Senior Engineer and higher for IC track), discuss justification/achievements
+    - Review list of people not progressing at all, people progressing significantly faster than others - discuss 
+  - For small families, it makes sense to do a combined session with members of 2 families together. The similar calibration sessions should also happen on higher levels:
+    - VP & Engineering Directors reviewing draft ratings of Engineering Managers
+    - CTO & VPs reviewing draft ratings of Engineering Directors
+
```
Comment (siroky) [inline tech/career-framework/calibration.md:34]: This blank line IMHO breaks formatting

File: tech/career-framework/readme.md [hunk 9]
```diff
@@ -30,3 +31,21 @@ There are several important events during the year that are focused on career an
 - **December** - [Performance Review 1/2](performance-reviews.md)
 
 The performance reviews are being done in a top-down manner to keep the expectations and requirements as synchronized as possible among the orgchart branches. Therfore we allocate two months for the performance reviews. Usually the management layers are done during the first month, individual contributor performance reviews the second month.
+
+## Timeline
+
+There are a lot of activities happenning during each performance review cycle, the table below highlights the most important ones:
+
+| Duration  | Activity |
```
Comment (siroky) [inline tech/career-framework/readme.md:39]: I'd update this timeline to be clear on what happens when. We have 2 months for performance reviews, so let's treat it as 8 weeks. And let's organize this according to what happens on which week, so that we can have clear timeline with deadlines and milestones. E.g. something like this:

- week 1 - manager training
- week 2-3 - self evaluation
- week 2-3 - manager evaluation
- week 4 - calibration session
- week 5-8 - review meetings
- week 7-8 - HR updates
- week 8 - ...

**Review by siroky — CHANGES_REQUESTED** [review]
In terms of contents, LGTM. I suggest to polish the calibration file a bit before merging. There are few formatting issues (missing dots at the ends of sentences, broken lists) and maybe several grammatical issues, so let's iron that out e.g. using Grammarly.

File: tech/career-framework/calibration.md (hunk 4 above)
Comment (susilm) [reply tech/career-framework/calibration.md]: makes sense. I've changed it for `Calibration Trainings`

File: tech/career-framework/calibration.md (hunk 6 above)
Comment (susilm) [reply tech/career-framework/calibration.md]: Rephrased, hopefully it makes more sense now.

File: tech/career-framework/readme.md (hunk 9 above)
Comment (susilm) [reply tech/career-framework/readme.md]: Good point, I've changed the format.

File: tech/career-framework/calibration.md (hunk 7 above)
Comment (susilm) [reply tech/career-framework/calibration.md:25]: Not sure where exactly you are recommending to have more info about it. Could you point me to a specific document/chapter please?

Comment (siroky) [reply tech/career-framework/calibration.md:25]: I'll be doing some further updates, e.g. related to the technical leadership involvement we discussed on Monday after this is merged so I'll tackle it as part of that. Basically I plan to go through the whole framework and interlink it / reference it where it makes sense.

**Review by siroky — APPROVED** [review]
